src/pages/Search/AdvancedSearchFilters.tsx

```diff
@@ -243,88 +243,6 @@ function AdvancedSearchFilters() {
-    //             route: ROUTES.SEARCH_ADVANCED_FILTERS_REPORT_ID,
-    //         },
-    //         {
-    //             title: getFilterDisplayTitle(searchAdvancedFilters, CONST.SEARCH.SYNTAX_FILTER_KEYS.AMOUNT, translate),
-    //             description: 'common.total' as const,
-    //             route: ROUTES.SEARCH_ADVANCED_FILTERS_AMOUNT,
-    //         },
-    //         {
-    //             title: getFilterDisplayTitle(searchAdvancedFilters, CONST.SEARCH.SYNTAX_FILTER_KEYS.CATEGORY, translate),
-    //             description: 'common.category' as const,
-    //             route: ROUTES.SEARCH_ADVANCED_FILTERS_CATEGORY,
-    //         },
-    //         {
-    //             title: getFilterDisplayTitle(searchAdvancedFilters, CONST.SEARCH.SYNTAX_FILTER_KEYS.KEYWORD, translate),
-    //             description: 'search.filters.hasKeywords' as const,
-    //             route: ROUTES.SEARCH_ADVANCED_FILTERS_KEYWORD,
-    //         },
-    //         {
-    //             title: getFilterCardDisplayTitle(searchAdvancedFilters, cardList),
-    //             description: 'common.card' as const,
-    //             route: ROUTES.SEARCH_ADVANCED_FILTERS_CARD,
-    //             shouldHide: Object.keys(cardList).length === 0,
-    //         },
-    //         {
-    //             title: getFilterTaxRateDisplayTitle(searchAdvancedFilters, taxRates),
-    //             description: 'workspace.taxes.taxRate' as const,
-    //             route: ROUTES.SEARCH_ADVANCED_FILTERS_TAX_RATE,
-    //         },
-    //         {
-    //             title: getFilterExpenseDisplayTitle(searchAdvancedFilters, translate),
-    //             description: 'search.expenseType' as const,
-    //             route: ROUTES.SEARCH_ADVANCED_FILTERS_EXPENSE_TYPE,
-    //         },
-    //         {
-    //             title: getFilterDisplayTitle(searchAdvancedFilters, CONST.SEARCH.SYNTAX_FILTER_KEYS.TAG, translate),
-    //             description: 'common.tag' as const,
-    //             route: ROUTES.SEARCH_ADVANCED_FILTERS_TAG,
-    //         },
-    //         {
-    //             title: getFilterParticipantDisplayTitle(searchAdvancedFilters.from ?? [], personalDetails),
-    //             description: 'common.from' as const,
-    //             route: ROUTES.SEARCH_ADVANCED_FILTERS_FROM,
-    //         },
-    //         {
-    //             title: getFilterParticipantDisplayTitle(searchAdvancedFilters.to ?? [], personalDetails),
-    //             description: 'common.to' as const,
-    //             route: ROUTES.SEARCH_ADVANCED_FILTERS_TO,
-    //         },
-    //         {
-    //             title: getFilterInDisplayTitle(searchAdvancedFilters, translate, reports),
-    //             description: 'common.in' as const,
-    //             route: ROUTES.SEARCH_ADVANCED_FILTERS_IN,
-    //         },
-    //     ],
-    //     [searchAdvancedFilters, translate, cardList, taxRates, personalDetails, reports],
-    // );
```

Comment:
Hmm, what is happening here? 🤔

Reply:
Ah yes, like I mentioned above this is a leftover after my previous PR; I was resolving conflicts at the Friday evening and left it by mistake so I'm deleting it here as the first occasion 😅